### ISSUE TITLE
Release 4.3.0 - Release Candidate 7 - Docker Listener Module testing

### ISSUE THREAD

**Issue by CarlosRS9** [wazuh/wazuh#13332]
# Docker Listener module testing

## Modules tests information
|                                  |                                             |
|----------------------------------|---------------------------------------------|
| **Main module tests issue**      | #13325  |
| **Version**                      | 4.3.0                                       |
| **Release candidate #**          | RC7                                         |
| **Tag**                          | https://github.com/wazuh/wazuh/tree/v4.3.0-rc5 |
| **Previous modules tests issue** | #12529 |

## Test report procedure

All test results must have one the following statuses: 
|                                  |                                                        |
|-----------------|-------------------------------------------------------------------------|
| :green_circle:  | The test passed successfully. 										    |
| :yellow_circle: | The test passed but some enhancements need to be applied to the module. |
| :red_circle:    | The test failed.                                                        |
| :white_circle:  | The test does not apply to the given bucket type or service.            |

Any failing test must be properly addressed with a new issue, detailing the error and the possible cause. It must be included in the [Release 4.3.0 project](https://github.com/orgs/wazuh/projects/4/) for triage.

The data available in our Wazuh-dev environment for each of these supported services will be used. Additional logs may be created if they are necessary or there is insufficient data for testing.

## Test results

|Test          | Status                  | Issue ref. |
|--------------------------------|-------------------------|------------|
| Test docker listener using ossec.conf | :green_circle:| -          |

## Auditors validation
The definition of done for this one is the validation of the conclusions and the test results from all auditors.

All checks from below must be accepted in order to close this issue.

- [x] @davidjiglesias
- [x] @CarlosRS9

**Comment by CarlosRS9** [comment]
## Tests results

Manual testing was performed to ensure everything works after the change. The tests were carried out using both a Wazuh manager and a Wazuh agent both using the following configuration:
```
<wodle name="docker-listener">
    <interval>30s</interval>
    <attempts>5</attempts>
    <run_on_start>yes</run_on_start>
    <disabled>no</disabled>
</wodle>
```

The [logall_json option](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#logall-json) must be enabled in the `ossec.conf` file before starting the tests.

### Test docker listener using ossec.conf in a manager
Restart Wazuh and check ossec.log to verify the module is running


<details><summary>ossec.log output</summary>


</details>



Test docker listener using ossec.conf in an agent
Restart Wazuh and check ossec.log to verify the module is running


<details><summary>ossec.log output</summary>


</details>

#### Conclusions


### Test docker listener using ossec.conf in an agent

Restart Wazuh and check ossec.log to verify the module is running


<details><summary>ossec.log output</summary>


</details>



Test docker listener using ossec.conf in an agent
Restart Wazuh and check ossec.log to verify the module is running


<details><summary>ossec.log output</summary>


</details>

#### Conclusions
